library/L2_HAL/displays/lcd/st7066u.hpp

```diff
@@ -167,56 +143,46 @@ class St7066u
   ///
   /// @param operation Operation transfer type.
   /// @param nibble    4-bit data to transfer.
-  void WriteNibble(WriteOperation operation, uint8_t nibble)
+  void WriteNibble(WriteOperation operation, uint8_t nibble) const
   {
-    kControlPins.e.Set(sjsu::Gpio::State::kHigh);
-    kControlPins.rs.Set(sjsu::Gpio::State(operation));
-    kControlPins.rw.Set(sjsu::Gpio::State::kLow);
-    // set nibble on 4-bit data bus
-    kControlPins.d7.Set(sjsu::Gpio::State((nibble >> 3) & 0x01));
-    kControlPins.d6.Set(sjsu::Gpio::State((nibble >> 2) & 0x01));
-    kControlPins.d5.Set(sjsu::Gpio::State((nibble >> 1) & 0x01));
-    kControlPins.d4.Set(sjsu::Gpio::State((nibble >> 0) & 0x01));
+    kEnablePin.Set(sjsu::Gpio::State::kHigh);
```

Comment:
```suggestion
    kEnablePin.SetHigh();
```
Just an FYI, may be a bit more concise if you simply use the helper functions for this.